app/views/compute_resources/form/_gce.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= textarea_f f, :password, :label => _("JSON certificate"),
+              :help_inline => file_field(:dd, :json, accept: "application/JSON", onchange: 'tfm.gce.jsonLoader(this)'),
```

Comment:
I tried to use react here for the file loader, but it didn't end well, so I switched it into the browser HTML5 native `file_field`. the button looks a bit off, but it's better than c&p the json content manualy :)